client/src/plugins/http/index.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+import interceptors from './interceptors'
+import { apiUrl } from '../../config'
+
+export const http = axios.create({
```

Comment:
Please include a comment explaining this can also be used from a simple import not only as a plugin. Thanks!